sidebars.js

```diff
@@ -382,6 +382,7 @@ module.exports = {
         "concepts/workflows",
         "concepts/activities",
         "temporal-explained/task-queues-and-workers",
+        "concepts/metrics",
```

Comment:
```suggestion
      "reference/worker-metrics",
```

Mind if we move this file to the `/docs/reference` directory?
And to be honest, this is a bit out of place for the concepts section - 
However there are other places we can link to it such as the "What is a Worker Entity" page -
We are aiming to do a revamp of the landing page soon that should highlight references like this.

Reply:
Done